### PR TITLE
Update to v1.3.0

### PR DESCRIPTION
**Pull request by xylar**
This version drops python 2.7 support.

Main packages updated:
 - mpas-analysis 1.2.6
 - e3sm_diags 2.0.0
 - processflow 2.2.3
 - cdat 8.2
- ilamb 2.5
 - geometric_features 0.1.4
 - mpas_tools 0.0.4
 - ncl >=6.6.2
 - mpi4py pointing to system MPI

### CODE REVIEW

**Comment by rpwagner** [comment]
@xylar if you want to trigger Travis CI on this, you may need to do a minor commit to the from branch. It's of limited use until we address #58, though.

**Comment by xylar** [comment]
@rpwagner, this branch is very much a work in progress so I'm not worried about re-triggering CI.  That'll happen naturally when I make changes here.

I'll update CI to use python 3 and drop the pinning on e3sm_unified as soon as I've uploaded new versions.  That will at least be a back-stop on #58 for now.

**Comment by rpwagner** [comment]
Makes sense. I'll stop changing things for now.

**Comment by sterlingbaldwin** [comment]
The current version of processflow is 2.2.3.

**Comment by chengzhuzhang** [comment]
We will use updated version of output_viewer 1.3.1

**Comment by xylar** [comment]
> We will use updated version of output_viewer 1.3.1

Sorry about that!  I missed that update, I guess.  Just updated and I'm testing a build now.

**Comment by xylar** [comment]
> The current version of processflow is 2.2.3.

@sterlingbaldwin, that's good to know, but I'm assuming you might need to have some testing and a new release once e3sm_diags and mpas-analysis are updated.  Or do you think it'll just work?

**Comment by sterlingbaldwin** [comment]
Im going to need to do testing, and if there're problems Im going to have to release a new version, so you might end up using 2.2.4.

**Comment by xylar** [comment]
@sterlingbaldwin, I'm currently not able to install e3sm_diags (because we've been waiting on a CDAT release) with the rest of e3sm-unified so I've been holding of on that one.  Obviously, I can't install processflow without e3sm_diags so I've been holding off on that, too, for now.  That's why I haven't bothered to put in a version of processflow yet.

**Comment by xylar** [comment]
@sterlingbaldwin, are you able to test with a local build of e3sm_diags using this PR?
https://github.com/E3SM-Project/e3sm_diags/pull/256
First, do a local build:
In the `meta.yaml`, change the path to `..`, since the release hasn't been made yet.  Then, run:
```
conda build -c conda-forge -c cdat/label/v82 -c e3sm meta.yaml
```
If all goes well, you can do:
```
conda create -n processflow_test -c conda-forge -c cdat/label/v82 -c e3sm python=3.7 e3sm_diags=2.0.0 mpas-analysis=1.2.5 ...
```
I haven't got the go-ahead yet to release `mpas-analysis` 1.2.6 so that will need to wait.  I don't think you'll have trouble with the latest release, since nothing big has changed about the config files.

**Comment by xylar** [comment]
@rpwagner, I updated the definition of the container and CI is failing (timed out in one case, can't find `singularity` in the other.  Could you have a look?  It may be better to address this in another PR unless it's easy.

**Comment by rpwagner** [comment]
@xylar sure, I'll look shortly.

File: e3sm-unified.def [hunk 1]
```diff
@@ -11,7 +11,7 @@ From: continuumio/miniconda
 
 %post -c /bin/bash
     . /opt/conda/etc/profile.d/conda.sh
-    conda create -n e3sm-unified-nox -c conda-forge -c e3sm -c cdat/label/v81 python=2.7 e3sm-unified=1.2.6 mesalib
+    conda create -n e3sm-unified-nox -c conda-forge -c e3sm -c cdat/label/v82 python=3.7 e3sm-unified=1.3.0 mesalib
```
Comment (rpwagner) [inline e3sm-unified.def]: @xylar The base image for this container is [miniconda](https://hub.docker.com/r/continuumio/miniconda). Can you try changing it to [miniconda3](https://hub.docker.com/r/continuumio/miniconda3)?
From this
```
From: continuumio/miniconda
```
to this
```
From: continuumio/miniconda3
```

**Comment by xylar** [comment]
Okay, I think CI is finally fixed.  @rpwagner, the fix was to add `travis_wait` so things don't time out.  I also pointed to a specific `singularity` tag rather than a release branch (which seems to be transient) and fixed a few other small issues.

**Comment by rpwagner** [comment]
All makes sense to me. Let me know if you need me to look at anything else.

**Pull request merged**